site/app/models/gradeable/Redaction.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace app\models\gradeable;
+
+use app\models\AbstractModel;
+use app\libraries\Core;
+
+/**
+ * Class Redaction
+ * @package app\models\gradeable
+ *
+ * All data describing the configuration of a gradeable redaction
+ *
+ * @method int getId()
+ * @method int getPageNumber()
+ * @method float getX1()
+ * @method float getY1()
+ * @method float getX2()
+ * @method float getY2()
+ *
+ */
+class Redaction extends AbstractModel implements \JsonSerializable {
+    /** @prop
+     * @var int The id of the redaction */
+    protected $id = -1;
+    /** @prop
+     * @var int The page number of the redaction */
+    protected $page_number = -1;
+    /** @prop
+     * @var float The x1 coordinate of the redaction */
+    protected $x1 = -1;
+    /** @prop
+     * @var float The y1 coordinate of the redaction */
+    protected $y1 = -1;
+    /** @prop
+     * @var float The x2 coordinate of the redaction */
+    protected $x2 = -1;
+    /** @prop
+     * @var float The y2 coordinate of the redaction */
+    protected $y2 = -1;
```

Comment:
There's no need to initialize these with a dummy value here given that you're already initializing them in the constructor.  Also, see above for my comment about moving this to Doctrine.